packages/@azure/servicebus/data-plane/test/stress/fixedNumberOfClientsMultipleQueues.ts

```diff
@@ -0,0 +1,62 @@
+import { ServiceBusClient, SendableMessageInfo } from "../../lib";
+
+const connectionString = "";
+
+const testDurationInMilliseconds = 60000 * 5 * 12 * 24 * 7; // 1 week
+
+const numOfClients = 10;
+
+let msgId = 1;
+
+let snapshotIntervalID: any;
+
+let isJobDone = false;
+
+async function main(): Promise<void> {
+  snapshotIntervalID = setInterval(snapshot, 5000); // Every 5 seconds
+  setTimeout(() => {
+    isJobDone = true;
+  }, testDurationInMilliseconds);
+
+  sendReceiveMessages();
+}
+
+async function sendReceiveMessages(): Promise<void> {
+  const ns = ServiceBusClient.createFromConnectionString(connectionString);
+
+  const clients = [];
+  for (let i = 0; i < numOfClients; i++) {
+    clients[i] = ns.createQueueClient(`t1-queue-new-${i + 1}`);
+  }
+
+  try {
+    while (!isJobDone) {
+      for (let i = 0; i < numOfClients; i++) {
+        const sender = clients[i].createSender();
+        const message: SendableMessageInfo = {
+          messageId: msgId,
+          body: "test",
+          label: `${msgId}`
+        };
+        msgId++;
+        await sender.send(message);
+        const receiver = clients[i].createReceiver();
+        const messagesReceived = await receiver.receiveBatch(1);
+        messagesReceived[0].complete();
+      }
+    }
+  } finally {
+    ns.close();
```

Comment:
This should be awaited as well. Same for other files

Reply:
Got it.
Since this is at the end of test run, it seems fine to await to handle the promise correctly. For other occurrences seems like .catch() might be better (to stay consistent with intent to not wanting to wait).